Dockerfile

```diff
@@ -79,7 +80,6 @@ RUN initdb -D /opt/RDKit-build/pgdata \
   && pg_ctl -D /opt/RDKit-build/pgdata stop
 
 
-ARG postgres_image_version=16.2
```

Comment:
There's no need to set it again here since they go out of scope when the next FROM block starts.

Reply:
good to know, thanks!